services/etl/dags/create_table_waqi_readings.py

```diff
@@ -0,0 +1,21 @@
+import pendulum
```

Comment:
you like the pendulum library? I can add it to the airnow dag to keep things consistent

Reply:
yeah let's keep pendulum, airflow docs recommend